docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -235,6 +235,9 @@ Locales may change between versions, so keeping track of your customized locales
 
 To add a custom Readme, add a markdown formatted file (without an `.md` extension) to `custom/options/readme`
 
+**NOTE:** readme templates support **variable expansion**.  
+currently there are `{Name}`, `{Description}`, `{CloneURL.SSH}` and `{CloneURL.HTTPS}`
```

Comment:
Maybe a small clarification of what name this is -- is it repo name or username or full name?

Reply:
@mrsdizzie in the context of repo name?

Reply:
added as we discust on discord